modules/installation-special-config-kmod.adoc

```diff
@@ -278,7 +278,7 @@ when you build your Ignition config.
 This must include new kernel packages as they are needed to match newly installed kernels.
 
 [id="provision-kernel-modules-via-machineconfig_{context}"]
-=== Provision kernel modules via a `MachineConfig` object
+=== Provision kernel modules via a `MachineConfig object
```

Comment:
Missing close quote.

Reply:
Oops, thanks! (I removed the open quote as well here as part of our docs guidelines--[no markup for headings](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#:~:text=as%20a%20heading.-,Do%20not%20use%20backticks%20or%20other%20markup%20in%20assembly%20or%20module%20headings.,-Discrete%20headings).)